csharp/extractor/Semmle.Extraction.CSharp.Standalone/CsProjFile.cs

```diff
@@ -121,18 +131,18 @@ private void ReadProjectFileAsXml(FileInfo filename)
             }
         }
 
-        string[] references;
-        string[] csFiles;
+        string[]? references;
+        string[]? csFiles;
 
         /// <summary>
         /// The list of references as a list of assembly IDs.
         /// </summary>
-        public IEnumerable<string> References => references;
+        public IEnumerable<string> References => references!;
```

Comment:
The compiler is not checking if fields are set inside methods called from the constructors.